feature-detects/css/remunit.js

```diff
@@ -25,7 +25,7 @@ define(['Modernizr', 'createElement'], function( Modernizr, createElement ) {
     try {
       div.style.fontSize = '3rem';
     }
-    catch( er ) {}
+    catch ( er ) {}
```

Comment:
Yep let's set that to true as we're pretty inconsistent on that front looking at the files.